frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/review-and-create-step.tsx

```diff
@@ -35,12 +45,27 @@ export const ReviewAndCreate: React.FC<ReviewAndCreateProps> = ({
       </TextContent>
       <dl>
         <ReviewListTitle text="Storage and nodes" />
-        <ReviewListBody noValue={nodes.length < MINIMUM_NODES || !scName}>
+        {stretchClusterChecked && (
+          <ReviewListBody noValue={!selectedArbiterZone}>
+            <div>
+              <p>
+                Arbiter zone:&nbsp;
+                <span className="text-muted">{selectedArbiterZone ?? 'None'}</span>
+              </p>
+            </div>
```

Comment:
```suggestion
              <p>
                Arbiter zone:&nbsp;
                <span className="text-muted">{selectedArbiterZone ?? 'None'}</span>
              </p>
```
I don't see the need for a div tag.